datasets/fake_news_english/fake_news_english.py

```diff
@@ -0,0 +1,101 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Fake News vs Satire: A Dataset and Analysis."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import json
+import os
+import pandas as pd
+import datasets
+import pandas._libs.lib as lib
+
+_CITATION = """
+@inproceedings{inproceedings,
+author = {Golbeck, Jennifer and Everett, Jennine and Falak, Waleed and Gieringer, Carl and Graney, Jack and Hoffman, Kelly and Huth, Lindsay and Ma, Zhenya and Jha, Mayanka and Khan, Misbah and Kori, Varsha and Mauriello, Matthew and Lewis, Elo and Mirano, George and IV, William and Mussenden, Sean and Nelson, Tammie and Mcwillie, Sean and Pant, Akshat and Cheakalos, Paul},
+year = {2018},
+month = {05},
+pages = {17-21},
+title = {Fake News vs Satire: A Dataset and Analysis},
+doi = {10.1145/3201064.3201100}
+}
+"""
+
+_DESCRIPTION = """
+Fake news has become a major societal issue and a technical chal- lenge for social media companies 
+to identify. This content is difficult to identify because the term "fake news" covers intentionally 
+false, deceptive stories as well as factual errors, satire, and sometimes, stories that a person just 
+does not like. Addressing the problem requires clear de nitions and examples. In this work, we present 
+a dataset of fake news and satire stories that are hand coded, verified, and, in the case of fake news, 
+include rebutting stories. We also include a thematic content analysis of the articles, identifying 
+major themes that include hyperbolic support or condemnation of a gure, conspiracy theories, racist 
+themes, and discrediting of reliable sources. In addition to releasing this dataset for research use, 
+we analyze it and show results based on language that are promising for classi cation purposes. 
+Overall, our contribution of a dataset and initial analysis are designed to support future work by 
+fake news researchers.
+"""
+
+_HOMEPAGE = "https://dl.acm.org/doi/10.1145/3201064.3201100"
+
+#_LICENSE = ""
+
+_URLs = "https://github.com/jgolbeck/fakenews/raw/master/FakeNewsData.zip"
+    
+class FakeNewsEnglish(datasets.GeneratorBasedBuilder):
+    """Fake News vs Satire: A Dataset and Analysis"""
+
+    VERSION = datasets.Version("1.1.0")
+
+    def _info(self):
+        features = datasets.Features(
+                {
+                    "url_of_article": datasets.Value("string"),
+                    "fake_or_satire": datasets.Value("string"),
```

Comment:
Seems this could be a `ClassLabel` feature: https://huggingface.co/docs/datasets/package_reference/main_classes.html#datasets.ClassLabel

You will need to regenerate the `dataset_infos.json` file after this change

Reply:
Indeed,

For example
```
"fake_or_satire": datasets.ClassLabel(names=["Satire", "Fake"])
```